karma.conf.js

```diff
@@ -0,0 +1,27 @@
+process.env.CHROME_BIN = require('puppeteer').executablePath()
+
+module.exports = function (config) {
+    config.set({
+        frameworks: ["mocha", "karma-typescript"],
+        files: [
+            'test/**/*.spec.ts',
+            'test/scenes/*.ts',
+            'src/**/*.ts'
+        ],
+        preprocessors: {
+            "**/*.ts": ["karma-typescript"]
+        },
+        reporters: ["spec", "junit", "karma-typescript"],
+        browsers: ["ChromeHeadless"],
+        junitReporter: {
+            outputDir: process.env.JUNIT_REPORT_PATH,
+            outputFile: process.env.JUNIT_REPORT_NAME,
+            useBrowserName: false
+        },
+        client: {
+            mocha: {
+                timeout: 0
```

Comment:
Reminding myself to possibly remove this when done. Was needed initially when figuring out the right time to grab game object references but I've now hooked onto the `postBoot` hook.